### PR TITLE
ci: Fix updater workflow

### PR DESCRIPTION
**Pull request by tim77**

### CODE REVIEW

**Comment by tim77** [comment]
https://github.com/flathub/flatpak-builder-lint/pull/69

File: .github/workflows/auto-update.yml [hunk 1]
```diff
@@ -3,36 +3,41 @@ name: Auto update
...
     steps:
       - name: Install dependencies
         run: |
           set -e
-          sudo apt install python3-toml python3-aiohttp
+          sudo apt install \
+            python3-aiohttp \
+            python3-toml
 
       - uses: actions/checkout@v2
```
Comment (gasinvein) [inline .github/workflows/auto-update.yml]: Let's try copying clone job from Code-OSS, maybe the last `git push` will stop failing then.
```suggestion
      - name: Checkout repository
        uses: actions/checkout@v3
        if: github.event_name == 'pull_request'
        with:
          ref: ${{ github.event.pull_request.head.ref }}
          submodules: true
```

File: update-generated-sources.sh [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    jq -r -e '.sources[] | objects | select(.type=="patch") | .path // .paths[]' \
+    <<<"$MODULE_OBJ"
+)
+
+if [ -z "${PYTHONPATH}" ]; then
+    export PYTHONPATH="$TOOLS_DIR/node"
+else
+    export PYTHONPATH="$TOOLS_DIR/node:${PYTHONPATH}"
+fi
+
```
Comment (gasinvein) [inline update-generated-sources.sh]: Leftover from Code-OSS update script, forgot to remove it.
```suggestion
```

File: .github/workflows/auto-update.yml [hunk 3]
```diff
@@ -3,36 +3,41 @@ name: Auto update
...
+          EMAIL: ${{ github.actor }}@users.noreply.github.com
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GIT_LFS_SKIP_SMUDGE: '1'
         run: |
           set -e
-          git config user.email "${GITHUB_ACTOR}@users.noreply.github.com"
-          git config user.name "Workflow trigger"
-          ./update.py veloren-git.json -g builder-tools/cargo/flatpak-cargo-generator.py
+          ./update-generated-sources.sh
           git push https://${GITHUB_ACTOR}:${{ secrets.GITHUB_TOKEN }}@github.com/${GITHUB_REPOSITORY}
```
Comment (gasinvein) [inline .github/workflows/auto-update.yml]: I believe we shouldn't need to do push over http
```suggestion
          git push
```

**Pull request merged**